app/mailers/spree/shipment_mailer.rb

```diff
@@ -2,8 +2,9 @@
 
 module Spree
   class ShipmentMailer < BaseMailer
-    def shipped_email(shipment, resend = false)
+    def shipped_email(shipment, delivery = true, resend = false)
```

Comment:
I noticed that you didn't need to change any code for moving the `resend` parameter. Searching further it seems that we are not using it. Let's get rid of it then.

Further, the `delivery` parameter doesn't need to be optional. We don't need to be backwards compatible here.

And finally, boolean flags can be confusing when you just read code like `shipped_email(shipment, false)` because the reader doesn't know what the `false` means without looking up the method definition. That's why Ruby has [keyword arguments](https://thoughtbot.com/upcase/videos/ruby-keyword-arguments).
```suggestion
    def shipped_email(shipment, delivery:)
```